runtime/rasdump/javadump.cpp

```diff
@@ -1052,14 +1052,30 @@ JavaCoreDumpWriter::writeEnvironmentSection(void)
 	/* Write the running mode data */
 	_OutputStream.writeCharacters("1CIRUNNINGAS   Running as ");
 
+	/* If JITServer enabled, print running mode as JITServer and client UID*/
+#ifdef JITSERVER_SUPPORT
```

Comment:
The way the code is structured right now, you cannot have an embedded JVM if we ever compile JITServer code in (i.e. if `JITSSERVER_SUPPORT` is enabled.
I am thinking that we don't want to run the client in the embedded mode, but only as a standalone JVM. Thus, maybe it's better to do:
```
#ifdef J9VM_BUILD_J2SE
#ifdef JITSERVER_SUPPORT
...
#else /* !JITSERVER_SUPPORT */
	_OutputStream.writeCharacters("a standalone");
#endif 
#else /* !J9VM_BUILD_J2SE */
       _OutputStream.writeCharacters("an embedded");
#endif
```

Reply:
Thanks, this has been fixed. When both `J9VM_BUILD_J2SE` and `JITSERVER_SUPPORT` are defined, the program will print JITServer mode.